MSAL/test/app/MSALTestAppSettings.m

```diff
@@ -62,7 +59,11 @@
 
 static NSArray<NSString *> *s_authorityTypes = nil;
 
-static NSDictionary *s_profiles = nil;
+static NSDictionary* s_additionalProfiles = nil;
```

Comment:
nit: please follow the coding style for the star position

Reply:
done